docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+Install from source (Windows)
+=========================================
+
+.. contents::
+
+This page describes the build and installation of PyBaMM from the source code, available on GitHub. Note that this is **not the recommended approach for most users** and should be reserved to people wanting to participate in the development of PyBaMM, or people who really need to use bleeding-edge feature(s) not yet available in the latest released version. If you do not fall in the two previous categories, you would be better off installing PyBaMM using pip or conda.
+
+Lastly, familiarity with the Python ecosystem is recommended (pip, virtualenvs).
+Here is a gentle introduction/refresher: `Python Virtual Environments: A Primer <https://realpython.com/python-virtual-environments-a-primer/>`_.
+
+
+Prerequisites
+---------------
+
+To use PyBaMM, you must have Python 3.9, 3.10, 3.11, or 3.12 installed.
+
+To install Python 3.X, download the installation files from `Python’s
+website <https://www.python.org/downloads/windows/>`_. Make sure to
+tick the box on ``Add Python 3.X to PATH``. For more detailed
+instructions please see the `official Python on Windows
+guide <https://docs.python.org/3.9/using/windows.html>`__.
+
+(Optional) If you wish to build the documentation locally, you can install ``graphviz`` using the `Chocolatey <https://chocolatey.org/>`_ package manager,
+
+.. code:: bash
+
+	choco install graphviz --version=8.0.5
+
+or follow the instructions on the `graphviz website <https://graphviz.org/download/>`_.
+
+To install PyBaMM, you will need to install the following components:
+
+C++ Compiler
+~~~~~~~~~~~~
+
+PyBaMM uses a recent version of Microsoft Visual C++ (MSVC), which you can get using `Build Tools for Visual Studio Code 2022 <https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022/>`_.
+
+Note that you won't need Visual Studio 2022 entirely; just ``Desktop development with C++`` will suffice.
+
+.. image:: images-windows-source/pic1-msvc-select.png
+	:height: 456px
+	:width: 810px
+	:scale: 100%
+	:alt: Image with the packages to install
+
+
+VCPKG
+~~~~~
+
+PyBaMM comes with a DAE solver based on the IDA solver provided by the SUNDIALS library.
+To use this solver, you must ensure you have the necessary SUNDIALS components
+installed on your system.
+
+VCPKG automatically installs them for you during the build process. To learn more, look at the `official documentation by Microsoft <https://learn.microsoft.com/en-us/vcpkg/get_started/overview/>`_.
+
+To install VCPKG, follow the steps below:
+
+1. Open a Command Shell, and navigate to ``C:\``.
+    * We are installing VCPKG in ``C:\vcpkg``
```

Comment:
I would change the recommended directory to be something more local.

Reply:
Usually VCPKG is installed in `C:\vcpkg`, `C:\Program Files\vcpkg` or `C:\Users\<username>\vcpkg`. All of these are good default system places.